src/com/facebook/buck/jvm/kotlin/KotlinTestDescription.java

```diff
@@ -122,36 +134,36 @@ public Arg createUnpopulatedConstructorArg() {
                     pathResolver,
                     params.getProjectFilesystem(),
                     args.resources),
-                defaultJavacOptions.getGeneratedSourceFolderName(),
-                /* proguardConfig */ Optional.empty(),
-                /* postprocessClassesCommands */ ImmutableList.of(),
-                /* exportDeps */ ImmutableSortedSet.of(),
-                /* providedDeps */ ImmutableSortedSet.of(),
+                templateJavacOptions.getGeneratedSourceFolderName(),
+                Optional.empty(),
+                ImmutableList.of(),
+                ImmutableSortedSet.of(),
+                ImmutableSortedSet.of(),
                 abiJarTarget,
                 JavaLibraryRules.getAbiInputs(resolver, testsLibraryParams.getDeps()),
-                /* trackClassUsage */ false,
-                /* additionalClasspathEntries */ ImmutableSet.of(),
+                false,
+                ImmutableSet.of(),
                 stepFactory,
-                /* resourcesRoot */ Optional.empty(),
-                /* manifest file */ Optional.empty(),
-                /* mavenCoords */ Optional.empty(),
-                /* tests */ ImmutableSortedSet.of(),
-                /* classesToRemoveFromJar */ ImmutableSet.of()
+                args.resourcesRoot,      /* resourcesRoot */
```

Comment:
@dsyang I did reintroduce these.  I actually to find them useful, where there might be a series of:
```java
Optional.empty(),
ImmutableList.of(),
ImmutableSortedSet.of(),
ImmutableSortedSet.of(),
```
It actually is useful to read:
```
Optional.empty(),        /* proguardConfig */
ImmutableList.of(),      /* postprocessClassesCommands */
ImmutableSortedSet.of(), /* exportedDeps */
ImmutableSortedSet.of(), /* providedDeps */
```